packages/forgae-cli/forgae-init/init.js

```diff
@@ -15,18 +15,24 @@
  *  PERFORMANCE OF THIS SOFTWARE.
  */
 const constants = require('./constants.json');
-const execute = require('./../utils').execute;
-const printError = require('./../utils').printError;
-const print = require('./../utils').print;
-const createMissingFolder = require('./../utils').createMissingFolder;
-const copyFileOrDir = require('./../utils').copyFileOrDir;
-const packageJson = require('../../package.json')
-const forgaeVersion = packageJson.version;
+const execute = require('forgae-utils').execute;
```

Comment:
const utils = require('forgae-utils');
const execute = utils.execute;
and so on ...